topologies/kne/README.md

```diff
@@ -0,0 +1,2 @@
+The otg version must be set according to the available release found [here](https://github.com/open-traffic-generator/ixia-c/releases)
```

Comment:
Can you expand the README to put the version setting in context of a short guide what needs to be done the first time someone tries to run an OTG test?

In particular:

 1. What are the prerequisites? Instructions for running Ixia-C or links to instructions?
 2. What are the files in this directory for?
 3. What changes need to be made locally? Be specific: each release has a versions.yaml file. Get the version from the "release" key and use it to update the version placeholder in the *.textproto file.
 4. Example command line.

Reply:
Updated README.md file